packages/styles/notice.css

```diff
@@ -33,6 +33,22 @@
   font-size: var(--text-size-smaller);
 }
 
+.Notice--default {
```

Comment:
`.Notice` _is_ the default, so I don't think we need an extra class here to define it as a variant.

Additionally, there's a border radius that needs to be included here.